docs/metrics/customizing-the-sdk/Program.cs

```diff
@@ -0,0 +1,61 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics.Metrics;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+public class Program
+{
+    private static readonly Meter MyMeter = new Meter("MyCompany.MyProduct.MyLibrary", "1.0");
+
+    public static void Main(string[] args)
+    {
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddSource(MyMeter.Name)
+            .AddView(instrumentName: "MyCounter") // MyCounter will be reported with defaults. This is done to ensure that any other wildcard Views for the same instrument does not affect this instrument.
```

Comment:
minor, these lines are too long, consider making them block comments, e.g.:

```csharp

using var meterProvider = Sdk.CreateMeterProviderBuilder()
    .AddSource(MyMeter.Name)
    /* MyCounter will be reported with defaults.
       This is done to ensure that any other wildcard Views for
       the same instrument does not affect this instrument. */
    .AddView(instrumentName: "MyCounter") 
```

Reply:
Agree.
Also I plan to move most of these comments to the readme, to make the program cleaner.
A readme section showing View examples should be very useful.